src/index.ts

```diff
@@ -89,3 +110,16 @@ export function geometryToCells(geometry, resolution: bigint): Quadbin[] {
     max_zoom: zoom
   }).map(([x, y, z]) => tileToCell({x, y, z}));
 }
+
+export function quadbinToBoundary(cell: bigint) {
+  const bbox = cellToBoundingBox(cell);
+  const boundary = [
+    [bbox[0], bbox[3]],
+    [bbox[0], bbox[1]],
+    [bbox[2], bbox[1]],
+    [bbox[2], bbox[3]],
+    [bbox[0], bbox[3]]
+  ];
+
+  return {type: 'Polygon', coordinates: [boundary]};
```

Comment:
I assume the output should be a GeoJson Polygon feature. If so could we add types? e.g. `Feature<Polygons>` from `@types/geojson`?

Reply:
Sure, that sounds great! I'll add this as a `devDependency` and will return a `Polygon` 👍